manifests/repose9.pp

```diff
@@ -0,0 +1,200 @@
+# include existing configuration files. The exact behavior is provider
+# dependent. Q.v.:
+# * Puppet type reference: {package, "purgeable"}[http://j.mp/xbxmNP]
+# * {Puppet's package provider source code}[http://j.mp/wtVCaL]
+# * System modifications (if any) will be reverted as good as possible
+# (e.g. removal of created users, services, changed log settings, ...).
+# * This is thus destructive and should be used with care.
+# Defaults to <tt>present</tt>.
+#
+# [*enable*]
+# Bool/String. Controls if the managed service shall be running(<tt>true</tt>),
+# stopped(<tt>false</tt>), or <tt>manual</tt>. This affects the service state
+# at both boot and during runtime.  If set to <tt>manual</tt> Puppet will
+# ignore the state of the service.
+# Defaults to <tt>true</tt>.
+#
+# [*autoupgrade*]
+# Boolean. If set to <tt>true</tt>, any managed package gets upgraded
+# on each Puppet run when the package provider is able to find a newer
+# version than the present one. The exact behavior is provider dependent.
+# Q.v.:
+# * Puppet type reference: {package, "upgradeable"}[http://j.mp/xbxmNP]
+# * {Puppet's package provider source code}[http://j.mp/wtVCaL]
+# Defaults to <tt>false</tt>.
+#
+# The default values for the parameters are set in repose::params. Have
+# a look at the corresponding <tt>params.pp</tt> manifest file if you need more
+# technical information about them.
+#
+# [*run_port*]
+# Integer.  The port the valve should run on
+# Defaults to <tt>9090</tt>
+#
+# [*daemon_home*]
+# String. Daemon home path
+# Defaults to <tt>/usr/share/lib/repose</tt>
+#
+# [*log_path*]
+# String. Log file directory path
+# Defaults to <tt>/var/log/repose</tt>
+#
+# [*pid_file*]
+# String. Pid file location
+# Defaults to <tt>/var/run/repose-valve.pid</tt>
+#
+# [*user*]
+# String. User to run the valve as
+# Defaults to <tt>repose</tt>
+#
+# [*daemonize*]
+# String. the path to the daemonize binary
+# Defaults to <tt>/usr/sbin/daemonize</tt>
+#
+# [*daemonize_opts*]
+# String. The daemonize options to be passed into daemonize.
+# DEPRECATED. This does nothing for repose 7+. This is replaced by a different
+# variable in the sysconfig file but we are not supporting overwriting it.
+# at this time.
+# Defaults to <tt>-c $DAEMON_HOME -p $PID_FILE -u $USER -o $LOG_PATH/stdout.log -e $LOG_PATH/stderr.log -l /var/lock/subsys/$NAME</tt>
+#
+# [*run_opts*]
+# String. The options sent to the run command.
+# DEPRECATED. This does nothing for repose 7+. This is replaced by a different
+# variable in the sysconfig file but we are not supporting overwriting it.
+# Defaults to <tt>-p $RUN_PORT -c $CONFIG_DIRECTORY</tt>
+#
+# [*java_options*]
+# String. Additional java options to pass to java_opts
+# NOTE: this also sets JAVA_OPTS for repose 7+
+# Defaults to <tt>undef</tt>
+#
+# [*saxon_home*]
+# String. Home directory for Saxon. Sets SAXON_HOME
+# Defaults to <tt>undef</tt>
+#
+# [*rh_old_packages*]
```

Comment:
Since this is repose 9 specific,  i'd say remove this and the next option.

Reply:
To handle backward compatibility this has to be here. The default is true - so we have to change it here. Also the namespaces option has to be set here as well so config files are written properly.